core/arch/arm/plat-imx/conf.mk

```diff
@@ -9,6 +9,7 @@ mx6d-flavorlist =
 mx6dl-flavorlist = mx6dlsabresd
 mx6s-flavorlist =
 mx7-flavorlist = mx7dsabresd
+mx7-flavorlist += mx7swarp7
```

Comment:
`mx7-flavorlist = mx7dsabresd mx7swrap7` is better.

Reply:
yes thanks I'll get to this tomorrow (today look at the time 02:58) :)

Reply:
done (completely forgot to update this before now - oops)